recipes/hyperion/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "0.9.7" %}
+
+package:
+  name: hyperion
+  version: {{version}}
+
+source:
+  fn: Hyperion-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/h/hyperion/Hyperion-{{version}}.tar.gz
+  md5: 9ab9f0341d83a2c7a1d996e7438ba8d0
+
+build:
+  number: 0
+  script:
+     - python setup.py install
+     - ./configure --prefix=$PREFIX
+     - make
+     - make install
+
+requirements:
+
+  build:
+    - python
+    - setuptools
+    - hdf5
+    - mpich
+    - numpy x.x
+    - gcc  # [osx]
```

Comment:
If you need this for `Fortran` than you need to add it for both Linux and OS X (`#  [now win]`) and you will need `libgfortran`  as a run time dependency, if you do not link against `libquadmath`, or `libgcc` on Linux if you do link against `libquadmath`. (Please do not ask why it is this mess :unamused:)